wsmaster/codenvy-hosted-report-shared/src/main/java/com/codenvy/report/shared/dto/Ip.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.report.shared.dto;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * @author Dmytro Nochevnov
+ */
+@DTO
+public interface Ip {
```

Comment:
It's just a value object which is in role of DTO now. So, I would prefer "Ip" name.